<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    font-family: &quot;Seagull&quot;;
    font-size: 12pt;
    color: rgb(255, 255, 255); 
}

QWidget#tab1, QWidget#tab2, QWidget#tab3 {
    background-color: #f8f9fa;
}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>550</width>
     <height>350</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Seagull</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
    background: #e1e4e8; 
    border: 1px solid #c1c4c8;
    padding: 12px 30px;
    margin-right: 2px;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
    color: #586069; 
}

QTabBar::tab:hover {
    background: #d1d5da;
}

QTabBar::tab:selected {
    background: #007bff; 
    color: #ffffff;  
    border-bottom-color: #007bff;
    font-weight: bold;
}

QTabWidget::pane {
    border: 2px solid #007bff;
    top: -2px;
    background: #ffffff;
    border-radius: 8px;
}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>Length</string>
    </attribute>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>485</width>
       <height>166</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSplitter {
    background-color: transparent;
    border: none;
}

QSplitter::handle {
    background-color: rgb(255, 255, 255)
}

QSplitter::handle:horizontal {
    width: 1px; 
}

QSplitter::handle:vertical {
    height: 1px;
}

QSplitter::handle:hover {
    background-color: #0078d4; 
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>90</number>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>70</number>
      </property>
      <widget class="QLineEdit" name="lineEdit1">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QLineEdit:hover {
    border: 1px solid #7fb3ff; 
}

QLineEdit:focus {
    border: 4px solid #0078d4;
    background-color: #ffffff;
}</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox1">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QComboBox:hover {
    border: 1px solid #7fb3ff; 
}

QComboBox:focus {
    border: 2px solid #0078d4; 
    background-color: #ffffff;
}</string>
       </property>
       <item>
        <property name="text">
         <string>meters (m)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kilometers (km)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>inches (in)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>feet (ft)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>miles (mi)</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>60</number>
      </property>
      <widget class="QLineEdit" name="lineEdit2">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QLineEdit:hover {
    border: 1px solid #7fb3ff; 
}

QLineEdit:focus {
    border: 4px solid #0078d4; 
    background-color: #ffffff;
}</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox2">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QComboBox:hover {
    border: 1px solid #7fb3ff; 
}

QComboBox:focus {
    border: 2px solid #0078d4; 
    background-color: #ffffff;
}</string>
       </property>
       <item>
        <property name="text">
         <string>meters (m)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kilometers (km)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>inches (in)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>feet (ft)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>miles (mi)</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>Mass</string>
    </attribute>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>485</width>
       <height>166</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSplitter {
    background-color: transparent;
    border: none;
}

QSplitter::handle {
    background-color: rgb(255, 255, 255); 
}

QSplitter::handle:horizontal {
    width: 1px; 
}

QSplitter::handle:vertical {
    height: 1px;
}

QSplitter::handle:hover {
    background-color: #0078d4; 
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>90</number>
     </property>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>70</number>
      </property>
      <widget class="QLineEdit" name="lineEdit3">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QLineEdit:hover {
    border: 1px solid #7fb3ff; 
}

QLineEdit:focus {
    border: 4px solid #0078d4;
    background-color: #ffffff;
}</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox3">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QComboBox:hover {
    border: 1px solid #7fb3ff; 
}

QComboBox:focus {
    border: 2px solid #0078d4; 
    background-color: #ffffff;
}</string>
       </property>
       <item>
        <property name="text">
         <string>kilograms (kg)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pounds (lb)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ounces (oz)</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>60</number>
      </property>
      <widget class="QLineEdit" name="lineEdit4">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QLineEdit:hover {
    border: 1px solid #7fb3ff; 
}

QLineEdit:focus {
    border: 4px solid #0078d4;
    background-color: #ffffff;
}</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox4">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QComboBox:hover {
    border: 1px solid #7fb3ff; 
}

QComboBox:focus {
    border: 2px solid #0078d4; 
    background-color: #ffffff;
}</string>
       </property>
       <item>
        <property name="text">
         <string>kilograms (kg)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pounds (lb)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ounces (oz)</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab3">
    <attribute name="title">
     <string>Temperature</string>
    </attribute>
    <widget class="QSplitter" name="splitter_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>485</width>
       <height>166</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSplitter {
    background-color: transparent;
    border: none;
}

QSplitter::handle {
    background-color: rgb(255, 255, 255); 
}

QSplitter::handle:horizontal {
    width: 1px; 
}

QSplitter::handle:vertical {
    height: 1px;
}

QSplitter::handle:hover {
    background-color: #0078d4; 
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>90</number>
     </property>
     <widget class="QSplitter" name="splitter_8">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>70</number>
      </property>
      <widget class="QLineEdit" name="lineEdit5">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QLineEdit:hover {
    border: 1px solid #7fb3ff; 
}

QLineEdit:focus {
    border: 4px solid #0078d4;
    background-color: #ffffff;
}</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox5">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QComboBox:hover {
    border: 1px solid #7fb3ff; 
}

QComboBox:focus {
    border: 2px solid #0078d4; 
    background-color: #ffffff;
}</string>
       </property>
       <item>
        <property name="text">
         <string>Celsius (째C)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fahrenheit (째F)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kelvin (K)</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_9">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>60</number>
      </property>
      <widget class="QLineEdit" name="lineEdit6">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QLineEdit:hover {
    border: 1px solid #7fb3ff; 
}

QLineEdit:focus {
    border: 4px solid #0078d4;
    background-color: #ffffff;
}</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox6">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: #ffffff; 
    border: 1px solid #a0c4ff; 
    border-radius: 5px;
    padding: 5px;
    color: #333333;
}

QComboBox:hover {
    border: 1px solid #7fb3ff; 
}

QComboBox:focus {
    border: 2px solid #0078d4; 
    background-color: #ffffff;
}</string>
       </property>
       <item>
        <property name="text">
         <string>Celsius (째C)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fahrenheit (째F)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kelvin (K)</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
